packages/gp2-components/src/organisms/__tests__/KeyInformationModal.test.tsx

```diff
@@ -111,6 +111,45 @@ describe('KeyInformationModal', () => {
     expect(screen.getByText(/Please add your city/i)).toBeVisible();
   });
 
+  it('does not call onSave with orcid when orcid is not provided', async () => {
```

Comment:
will that not be a problem if someone has an orcid but then wants to remove it?

Reply:
I've tested it here and it still works because the data provider deals with it here https://github.com/yldio/asap-hub/blob/master/apps/gp2-server/src/data-providers/user.data-provider.ts#L225:L242